model/qti/ImportService.php

```diff
@@ -434,6 +437,8 @@ public function importQtiItem(
         $itemMustBeOverwritten = false,
         &$overwrittenItems = []
     ) {
+        $lock = $this->createLock('lock_'.$qtiItemResource->getIdentifier());
```

Comment:
`\oat\taoQtiItem\model\qti\ImportService::importQtiItem() `method in 280 lines length and completely unreadable. Do you thinks it's possible to refactor it?